packages/main/src/features/InputSuggestions.js

```diff
@@ -292,6 +307,20 @@ class Suggestions {
 		this.responsivePopover = staticAreaItem.querySelector("ui5-responsive-popover");
 		return this.responsivePopover;
 	}
+
+	get itemSelectionAnnounce() {
+		const i18nBundle = this.i18nBundle,
+			itemPositionText = i18nBundle.getText(LIST_ITEM_POSITION, [this.accInfo.currentPos], [this.accInfo.listSize]),
+			itemSelectionText = i18nBundle.getText(LIST_ITEM_SELECTED);
+
+		return `${itemPositionText} ${this.accInfo.itemText} ${itemSelectionText}`;
+	}
+
+	static async onDefine() {
```

Comment:
InputSuggestions.js is not a web component (does not extend any class in fact) so this hook does nothing for it. It still works, because Input.js loads this anyway. Please remove `onDefine` and don't import `fetchI18nBundle` at all. Leave the part in the constructor as it is.

We'll make sure we improve this in the future to make this connection more explicit.

Reply:
Done.